packages/web/lib/logging/server.ts

```
@@ -2,30 +2,40 @@
 
-// Custom Sentry transport
-class SentryTransport extends winston.Transport {
-  log(info: winston.LogEntry, callback: () => void) {
-    setImmediate(() => {
-      this.emit('logged', info)
-    })
+// Custom Sentry transport using winston's Transform base class
+class SentryTransport extends Transform {
+  constructor(options?: any) {
+    super(options)
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)